serverless/serverless
# Update Azure provider guide docs

## What did you implement:

Updated Azure Guide Documentation to reflect current state of plugin



## How did you implement it:

Updated all docs



## How can we verify it:

Read through docs and make sure they make sense. Even better, create your own Azure Function App and follow along :shipit: 



## Todos:

_**Note: Run `npm run test-ci` to run all validation checks on proposed changes**_

       **Validate via `npm test`**
       **Validate via `npm run lint-updated`**  
       _Note: Some reported issues can be automatically fixed by running `npm run lint:fix`_
       **Validate via `npm run prettier-check-updated`**  
       _Note: All reported issues can be automatically fixed by running `npm run prettify-updated`_

**_Is this ready for review?:_** YES  
**_Is it a breaking change?:_** NO
